src/value.rs

```diff
@@ -435,6 +437,35 @@ where
     }
 }
 
+impl<'a> From<Cow<'a, str>> for Value {
+    fn from(x: Cow<'a, str>) -> Value {
+        // will stack overflow if not cloning the ref
+        #[allow(clippy::clone_double_ref)]
+        x.as_ref().clone().into()
```

Comment:
`x.into_owned().into()`? And you don`t need to skip clippy warning

Reply:
I simply did not know about it, will fix